backend/build.gradle.kts

```diff
@@ -50,7 +51,9 @@ dependencies {
     api("org.commonmark:commonmark:0.22.0")
     api("org.commonmark:commonmark-ext-gfm-tables:0.22.0")
     api("com.fasterxml.jackson.dataformat:jackson-dataformat-csv")
-    api("com.squareup.okhttp3:okhttp:4.12.0")
+    api("com.amazonaws:aws-java-sdk-cognitoidp:1.12.765")
```

Comment:
Are these libraries used by this feature?

Reply:
Eventually not, removed them